src/redux/books/booksSlice.js

```diff
@@ -1,42 +1,64 @@
-import { createSlice } from '@reduxjs/toolkit';
-import { v4 as id } from 'uuid';
+/* eslint-disable */
+
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
 
 const initialState = {
-  books: [
-    {
-      item_id: id(),
-      title: 'The Great Gatsby',
-      author: 'John Smith',
-      category: 'Fiction',
-    },
-    {
-      item_id: id(),
-      title: 'Anna Karenina',
-      author: 'Leo Tolstoy',
-      category: 'Fiction',
-    },
-    {
-      item_id: id(),
-      title: 'The Selfish Gene',
-      author: 'Richard Dawkins',
-      category: 'Nonfiction',
-    },
-  ],
+  books: [],
+  isLoading: false,
+  error: undefined,
 };
 
+const APP_ID = 'Cr1NFgvae4ISnLGHWK50';
+const BASE_URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/';
+const BOOKS_URL = `${BASE_URL}${APP_ID}/books`;
+
+export const createAppId = createAsyncThunk('books/createAppId', async () => {
+  try {
+    const response = await axios.post(`${BASE_URL}`);
+    return response.data;
+  } catch (error) {
+    return error.message;
+  }
+});
+
+export const addBooks = createAsyncThunk('books/addBooks', async (newBook) => {
+  await axios.post(`${BOOKS_URL}`, newBook);
+  return newBook;
+});
+
+export const getBooks = createAsyncThunk('books/getBook', async () => {
+  const response = await axios.get(`${BOOKS_URL}`);
+  return response.data;
+});
+
+export const removeBooks = createAsyncThunk('books/removeBooks', async (bookId) => {
+  await axios.delete(`${BOOKS_URL}/${bookId}`);
+  return bookId;
+});
+
 const bookSlice = createSlice({
   name: 'book',
   initialState,
-  reducers: {
-    addBook: (state, action) => {
-      state.books.push(action.payload);
-    },
-    removeBook: (state, action) => {
+  reducers: {},
+  extraReducers: (builder) => {
+    builder.addCase(addBooks.fulfilled, (state, action) => {
+      const newBook = action.payload;
+      state.books.push(newBook);
+    });
+    builder.addCase(getBooks.fulfilled, (state, action) => {
+      const booksData = action.payload;
+      const books = Object.entries(booksData).map(([item_id, bookArray]) => ({
+        ...bookArray[0],
+        item_id,
+      }));
```

Comment:
Hello code reviewers,
I've disabled eslint for this file because it's telling me to use camelCase for the item_id.